README.md

```diff
@@ -159,6 +159,38 @@ Trigger a visualization for the track. This is typically album art.
 
 **This is an untested method.**
 
+### Queue
+`gmusic.queue` exposes methods to query and control the current queue
+
+#### `queue.getSongs()`
+Retrieve the songs currently in the queue
+
+**Returns:**
+
+- retVal `Array` - An array of song objects representing the songs currently in the queue.  Song objects have the following structure
+
+```
+{
+  id: String,         // A unique ID for this song
+  title: String,      // The songs title
+  album: String,      // The songs album
+  artist: String,     // The songs artist
+  art: String,        // A URL to the album art for this song
+  playing: Boolean,   // True if the song is currently playing, false otherwise
+  duration: Number    // Duration of song in milliseconds
+}
+```
```

Comment:
In terms of generating a UI though, imo it is more logical to be able to say `getSongs()` and then render the one that has `playing: true` slightly differently.  Rather than having to do something like.

```js
var currentSong = getSong();
getSongs().forEach((song) => {
    if (song.id === currentSong.id) {
        renderCurrentSong(song);
    } else {
        renderNormalSong(song);
    }
});
```

That if check is extra work a implementing library shouldn't have to do IMO.  Adding it to `change:song` would still be logical (it would just always be `true`). Thoughts?